velox/exec/PartitionedOutput.h

```diff
@@ -72,6 +77,12 @@ class Destination {
   void
   serialize(const RowVectorPtr& input, vector_size_t begin, vector_size_t end);
 
+  void setTargetSizePct() {
```

Comment:
Would you document this method and explain why don't we flush at 100% all the time?